web/src/views/UserLoginView/UserLoginView.vue

```diff
@@ -1,41 +1,29 @@
 <template>
   <v-container v-page-title="'Log In'">
-    <GirderAuth
-      v-if="!DJANGO_API"
-      :force-otp="false"
-      :show-forgot-password="false"
-      :oauth="true"
-    />
-    <div v-else>
-      <p>
-        TODO: Implement OAuth
-      </p>
-      <p>
-        For now, you can "log in" to a Django account with a username/password
-        using this console command:
-        <code>
-          setTokenHack('django_auth_token_for_user')
-        </code>
-      </p>
-      <p>
-        You can create a bookmark with this URL to make this quicker:
-        <code>
-          javascript:setTokenHack('django_auth_token_for_user')
-        </code>
-      </p>
-    </div>
+    <p>
+      TODO: Implement OAuth
+    </p>
+    <p>
+      For now, you can "log in" to a Django account with a username/password
+      using this console command:
+      <code>
+        setTokenHack('django_auth_token_for_user')
+      </code>
+    </p>
+    <p>
+      You can create a bookmark with this URL to make this quicker:
+      <code>
+        javascript:setTokenHack('django_auth_token_for_user')
+      </code>
+    </p>
```

Comment:
This looks weird--can we file an issue to figure out whether this should be in the codebase at all? (attn: @dchiquito, @AlmightyYakob)

Reply:
I don't believe this view is naturally accessed at all anymore. The only way I was able to get to this page is to logout on the client, and then manually go to `/user/login`. I think we can remove this view and it's entry in the router. @dchiquito any objection? If not, I'll file an issue.

Reply:
Go ahead and file the issue and/or strip it out in a new PR.

Reply:
#788